cloneSTEPWAT2.sh

```diff
@@ -4,7 +4,7 @@
 cd R_program
 #module load git/2.17.1
 wait
-git clone --branch master --recursive https://github.com/DrylandEcology/STEPWAT2.git
+git clone --branch bugfix_337_reorder_species_input --recursive https://github.com/DrylandEcology/STEPWAT2.git
 wait
```

Comment:
@kpalmqui I just reset the STEPWAT2 branch to master and merged master into this branch so we should be ready to merge this branch into master. If you would like to test this branch before we merge make sure you change the branch back to bugfix_337_reorder_species_input because I haven't merged bugfix_337_reorder_species_input into STEPWAT2 master yet. 

I need to perform both merges on the same day, because unfortunately no matter how I merge them rSFSTEP2 master will be broken until both merges are complete.

Reply:
I tested STEPWAT2 standalone several times and everything looks good. Once we update the SOILWAT2 submodule link, we should be good to go.